### ISSUE TITLE
is_greater_than

### ISSUE THREAD

**Issue by andersonmdcanteli** [puzzle-in-a-mug/paramcheckup#4]
The function numbers.is_greater_than is not working as it should.

```from paramcheckup import numbers
numbers.is_greater_than(
    value=4,
    param_name="x_min",
    lower=4,
    inclusive=True,
    func_name="func_name",
)
```

Output:

`The value of the parameter 'x_min' in function 'func_name' must be equal or greater than '4', but it is '4'.`

**Comment by andersonmdcanteli** [comment]
Fixed in version 1.0.1

**Comment by andersonmdcanteli** [comment]
Fixed in version 1.0.1
